Makefile

```diff
@@ -0,0 +1,31 @@
+#
+#  Copyright (c) 2021 International Business Machines
+#  All rights reserved.
+#
+#  SPDX-License-Identifier: LGPL-3.0-or-later
+#
+#  Authors: anita.shekar@ibm.com, sandy.kaur@ibm.com
+#
+
+MODULE := control
+CONFIG ?= ceph-nvmeof.conf
+
+setup: requirements.txt
+	pip3 install -r requirements.txt
+
+grpc:
+	@python3 -m grpc_tools.protoc \
+			--proto_path=./$(MODULE)/proto \
+			--python_out=./$(MODULE)/proto \
+			--grpc_python_out=./$(MODULE)/proto \
```

Comment:
The generated grpc file should be stored not with the model but in a separate folder, we will probably make it a package to be used in other projects/repos

Reply:
For now I've placed the generated grpc files in the 'generated` folder within `control` so as to not complicate relative imports, but this is flexible.